src/Chunk.ts

```diff
@@ -158,23 +158,27 @@ export default class Chunk {
 				this.isManualChunk = true;
 			}
 			module.chunk = this;
-			if (module.isEntryPoint && !this.entryModule) {
-				this.entryModule = module;
-				this.isEntryModuleFacade = true;
+			if (module.isEntryPoint || (module.isDynamicEntryPoint && !inlineDynamicImports)) {
+				this.entryModules.push(module);
```

Comment:
This is something I should have done a long time ago, thanks, a much richer representation.